tests/conftest.py

```diff
@@ -139,47 +141,51 @@ def backend(request):
     await send({"type": "http.response.body", "body": json.dumps(body).encode()})
 
 
-class CAWithPKEncryption(trustme.CA):
-    """Implementation of trustme.CA() that can emit
-    private keys that are encrypted with a password.
-    """
+SERVER_SCOPE = "session"
 
-    @property
-    def encrypted_private_key_pem(self):
-        return trustme.Blob(
-            self._private_key.private_bytes(
-                Encoding.PEM,
-                PrivateFormat.TraditionalOpenSSL,
-                BestAvailableEncryption(password=b"password"),
-            )
-        )
 
+@pytest.fixture(scope=SERVER_SCOPE)
+def cert_authority():
+    return trustme.CA()
 
-SERVER_SCOPE = "session"
+
+@pytest.fixture(scope=SERVER_SCOPE)
+def ca_cert_pem_file(cert_authority):
+    with cert_authority.cert_pem.tempfile() as tmp:
+        yield tmp
 
 
 @pytest.fixture(scope=SERVER_SCOPE)
-def example_cert():
-    ca = CAWithPKEncryption()
-    ca.issue_cert("example.org")
-    return ca
+def localhost_cert(cert_authority):
+    return cert_authority.issue_cert("localhost")
 
 
 @pytest.fixture(scope=SERVER_SCOPE)
-def cert_pem_file(example_cert):
-    with example_cert.cert_pem.tempfile() as tmp:
+def cert_pem_file(localhost_cert):
+    with localhost_cert.cert_chain_pems[0].tempfile() as tmp:
```

Comment:
From the trustme docs: "The zeroth entry in this list is the actual PEM-encoded certificate..."

Just including this cert and not the whole chain is fine, since there are no intermediary CA certs.